platform-ui-skin/src/main/webapp/skin/less/variables.less

```diff
@@ -90,16 +90,16 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 // Accent colors
 // -------------------------
 @errorColor:        #bc4343;     // border error
-@errorColorLight:   #d07b7b;   // background error
```

Comment:
Why did you change these colors codes? they can be used by other applications.
if you want to use #ffffff color ya can use @baseBackground variable

Reply:
IMO it will be better to change these common alert colors so we will have alerts with the same colors on the whole platform